syntax = "proto3";
package BankService2;

service BankService2 {
    rpc MsgDelivery (MsgRequest) returns (MsgResponse) {}
    rpc MsgPropagation(MsgRequest) returns (MsgResponse) {}
}

message MsgRequest {
    int64 id = 1;
    int64 customer_request_id = 2;
    string interface = 3;
    float amount = 4;
    int64 clock = 5;
}

message MsgResponse {
    int64 id = 1;
    string interface = 2;
    string response = 3;
    float amount = 4;
    int64 clock = 5;
    int64 customer_request_id = 6;
}